src/LaraCart.php

```diff
@@ -731,13 +731,21 @@ public function getFees()
      * Gets the total amount discounted.
      *
      * @param bool $format
+     * @param bool $withItemDiscounts
      *
      * @return string
      */
-    public function totalDiscount($format = true)
+    public function totalDiscount($format = true, $withItemDiscounts = true)
```

Comment:
@lukepolo Shouldn't it be '$withItemDiscounts =  false' so that backwards compatibility is maintained?

Reply:
It was a bug, so anything backwards was actually broken

Reply:
talked further, will bump version